pkg/contractspecs/specs.go

```diff
@@ -60,8 +60,38 @@ var (
 	// <contract type name in lower>:<event name in lower>
 	// ex. newsroomcontract:watchcontentpublished.
 	// If true, disables tracking of that event
+	//
 	// Use FlagKey() to consistently create keys for this map
-	DisableCrawl = map[string]bool{}
+	//
+	// NOTE: The currently disabled events match up items Civil is not processing.
+	// Feel free to alter and update as needed.
+	// Should be a config file or env vars in the future
+	DisableCrawl = map[string]bool{
+		FlagKey("CivilTCRContract", "GovernmentTransfered"): true,
+
+		FlagKey("NewsroomContract", "ContentPublished"):   true,
+		FlagKey("NewsroomContract", "OwnershipRenounced"): true,
+		FlagKey("NewsroomContract", "RevisionSigned"):     true,
+		FlagKey("NewsroomContract", "RoleAdded"):          true,
+		FlagKey("NewsroomContract", "RoleRemoved"):        true,
+
+		FlagKey("CVLTokenContract", "OwnershipRenounced"):   true,
+		FlagKey("CVLTokenContract", "OwnershipTransferred"): true,
+		FlagKey("CVLTokenContract", "Approval"):             true,
+
+		FlagKey("ParameterizerContract", "RewardClaimed"): true,
```

Comment:
are you sure this isn't currently being processed? I'll need to check how param proposal rewards are being updated since it does seem as though user challenge data is updated after rewards are claimed

Reply:
when I put these into the configuration, I was checking it against what was being processed in the civil processor.  If you need to add this, then add the processor code and remove this exception.

https://github.com/joincivil/civil-events-processor/blob/master/pkg/processor/parameterizerProcessor.go#L107